services/black/src/commands/Validation/SetGuestInformationValidator.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace artiso.AdsdHotel.Black.Commands.Validation
+{
+    public static class SetGuestInformationValidator
+    {
+        public static bool IsValid(SetGuestInformation sgi)
```

Comment:
Suggestion: validate `GuestInformation`, not `SetGuestInformation` which is an internal component.